packages/cozy-harvest-lib/src/components/AccountFormError.jsx

```diff
@@ -0,0 +1,81 @@
+import React, { PureComponent } from 'react'
+import PropTypes from 'react-proptypes'
+
+import { translate } from 'cozy-ui/react/I18n'
+
+import { KonnectorJobError } from '../helpers/konnectors'
+import Markdown from './Markdown'
+
+/**
+ * Return the locale key corresponding to error description for a given error
+ * code.
+ * @param  {string} code Error code as returned by a konnector job
+ * @return {string}      locale key
+ */
+const getKonnectorJobKey = code => `error.job.${code}.description`
+
+/**
+ * Component used to show an error related to an account or to a konnector job
+ * Regular error are treated in a common way, but for KonnectorJobErrors
+ * we can add a little bit of logic.
+ *
+ * This component is strongly related with the content of locales files and
+ * deals mainly with translation concerns.
+ */
+export class AccountFormError extends PureComponent {
+  defaultKey = 'error.job.UNKNOWN_ERROR.description'
+
+  /**
+   * Returns the description of an error.
+   * This method first try to get the full error code
+   * (for example LOGIN_FAILED.LOGIN_FAILED.NEEDS_SECRET), then fallback on the
+   * error type, which is the error code first segment (in our example, it's
+   * LOGIN_FAILED). It none of these two tries returns anything, it means
+   * that the error is unknown or not yet handled by harvest, so we fallback
+   * to the default error messages.
+   * @param  {Error} error      The error
+   * @param  {Object} konnector konnector related to this error
+   * @param  {Func} t           Translation function, expected to be Polyglot.t()
+   * @return {String}           The error description
+   */
+  getDescription() {
+    const { error, konnector, t } = this.props
+
+    const args = {
+      name: konnector.name || '',
+      link: konnector.vendor_link || ''
+    }
+
+    if (error instanceof KonnectorJobError) {
+      return t(getKonnectorJobKey(error.code), {
+        ...args,
+        _: t(getKonnectorJobKey(error.type), {
```

Comment:
What is this `_` key?

Reply:
It allows to specify a fallback value is the key does not exist in translation file.
See (Polyglot documentation)[https://github.com/airbnb/polyglot.js?files=1#polyglotprototypetkey-interpolationoptions].

Here we are using this feature on three levels 💪, it spares us [this kind of mess](https://github.com/cozy/cozy-home/blob/master/src/lib/konnectors.js#L99)